error/src/lib.rs

```diff
@@ -19,162 +19,393 @@ use core::num::NonZeroU32;
+    pub const DRIVER_ECC384_READ_SEED_KV_READ: CaliptraError = CaliptraError::new_const(0x00050001);
+    pub const DRIVER_ECC384_READ_SEED_KV_WRITE: CaliptraError =
+        CaliptraError::new_const(0x00050002);
+    pub const DRIVER_ECC384_READ_SEED_KV_UNKNOWN: CaliptraError =
+        CaliptraError::new_const(0x00050003);
+
+    pub const DRIVER_ECC384_WRITE_PRIV_KEY_KV_READ: CaliptraError =
+        CaliptraError::new_const(0x00050004);
+    pub const DRIVER_ECC384_WRITE_PRIV_KEY_KV_WRITE: CaliptraError =
+        CaliptraError::new_const(0x00050005);
+    pub const DRIVER_ECC384_WRITE_PRIV_KEY_KV_UNKNOWN: CaliptraError =
+        CaliptraError::new_const(0x00050006);
+
+    pub const DRIVER_ECC384_READ_PRIV_KEY_KV_READ: CaliptraError =
+        CaliptraError::new_const(0x00050007);
+    pub const DRIVER_ECC384_READ_PRIV_KEY_KV_WRITE: CaliptraError =
+        CaliptraError::new_const(0x00050008);
+    pub const DRIVER_ECC384_READ_PRIV_KEY_KV_UNKNOWN: CaliptraError =
+        CaliptraError::new_const(0x00050009);
+
+    pub const DRIVER_ECC384_READ_DATA_KV_READ: CaliptraError = CaliptraError::new_const(0x0005000a);
+    pub const DRIVER_ECC384_READ_DATA_KV_WRITE: CaliptraError =
+        CaliptraError::new_const(0x0005000b);
+    pub const DRIVER_ECC384_READ_DATA_KV_UNKNOWN: CaliptraError =
+        CaliptraError::new_const(0x0005000c);
+
+    pub const DRIVER_KV_ERASE_USE_LOCK_SET_FAILURE: CaliptraError =
+        CaliptraError::new_const(0x00060001);
+    pub const DRIVER_KV_ERASE_WRITE_LOCK_SET_FAILURE: CaliptraError =
+        CaliptraError::new_const(0x00060002);
+
+    pub const DRIVER_PCR_BANK_ERASE_WRITE_LOCK_SET_FAILURE: CaliptraError =
+        CaliptraError::new_const(0x00070001);
+
+    /// Mailbox Errors
+    pub const DRIVER_MAILBOX_INVALID_STATE: CaliptraError = CaliptraError::new_const(0x00080001);
+    pub const DRIVER_MAILBOX_INVALID_DATA_LEN: CaliptraError = CaliptraError::new_const(0x00080002);
+    pub const DRIVER_MAILBOX_NO_DATA_AVAIL: CaliptraError = CaliptraError::new_const(0x00080003);
+    pub const DRIVER_MAILBOX_ENQUEUE_ERR: CaliptraError = CaliptraError::new_const(0x00080004);
+    pub const DRIVER_MAILBOX_DEQUEUE_ERR: CaliptraError = CaliptraError::new_const(0x00080005);
+
+    pub const DRIVER_SHA384ACC_INVALID_OP: CaliptraError = CaliptraError::new_const(0x00090001);
+    pub const DRIVER_SHA384ACC_MAX_DATA_ERR: CaliptraError = CaliptraError::new_const(0x00090002);
+    pub const DRIVER_SHA384ACC_INDEX_OUT_OF_BOUNDS: CaliptraError =
+        CaliptraError::new_const(0x00090003);
+
+    pub const DRIVER_SHA1_INVALID_STATE: CaliptraError = CaliptraError::new_const(0x000a0001);
+    pub const DRIVER_SHA1_MAX_DATA: CaliptraError = CaliptraError::new_const(0x000a0002);
+    pub const DRIVER_SHA1_INVALID_SLICE: CaliptraError = CaliptraError::new_const(0x000a0003);
+    pub const DRIVER_SHA1_INDEX_OUT_OF_BOUNDS: CaliptraError = CaliptraError::new_const(0x000a0004);
+
+    ///        ManifestMarkerMismatch = 1,
```

Comment:
Add comment here for `/// Image Verify Errors`? Also might recommend removing the comments above each error since they seem redundant.

Reply:
done!